pkg/process/config/yaml_config.go

```diff
@@ -78,7 +78,7 @@ func (a *AgentConfig) LoadProcessYamlConfig(path string) error {
 		if ok {
 			a.Enabled, a.EnabledChecks = true, processChecks
 		} else if enabled == "disabled" {
-			a.Enabled = false
+			a.Enabled, a.EnabledChecks = false, nil
```

Comment:
should this just be captured in an `else`? this would cover the case where there is a user typo like "false" or "disable"